databases/cockroachdb/README.md

```diff
@@ -155,105 +115,25 @@ DATABASE_URL="postgresql://root@localhost:26257/prisma?sslmode=disable"
 
 > **Note:** If you're using CockroachDB Serverless, see [`.env.example`](./.env.example) for more information on how `DATABASE_URL` should look like with the cluster configruation.
```

Comment:
It seems this file is missing – could you include it in the PR?

Reply:
This file isn't in the main repository either. 

![no file](https://user-images.githubusercontent.com/69592270/165003101-9edf1537-a903-492f-9286-5576205a1688.png)

I'll add something in, but I'm not sure if it'll be entirely accurate.

Reply:
Sounds good. 👍🏾